x/txfees/module.go

```diff
@@ -190,8 +190,6 @@ func (AppModule) ConsensusVersion() uint64 { return 1 }
 // If they have, we unmarshal the current consensus params, update the target gas, and cache the value.
 // This is done to improve performance by not having to fetch and unmarshal the consensus params on every block.
 // TODO: Move this to EIP-1559 code
-// UNFORKING v2 TODO: Do we still want to use cachedConsParams here? I guess it removes the need to do arithmetic operations on every block.
```

Comment:
I think it's fine as is, next step with this code is to remove in favor of consensus fee markets so no point in worrying here too much